### ISSUE TITLE
How to reset keras random state many times inside python?

### ISSUE THREAD

**Issue by farmeh** [fchollet/keras#3538]
For generating different initial random state for neural network I have been doing this based on 
suggestions in the forum:

import numpy as np ;
import datetime ; 
np.random.seed(datetime.datetime.now().microsecond)
from keras import ....

(setting numpy random seed before importing anything from keras). 

Now, assume that I want to train/evaluate a neural network with different initial random states, 
for 10 times, and take average of f-scores. I had to run the file externally for 10 times (e.i., python myprog.py) to make it work. 

Now assume I want to this inside python:

for i in range(10):
         import numpy as np ;
         import datetime ; 
         np.random.seed(datetime.datetime.now().microsecond)
         from keras import ....
         train()
         evaluate()

but I guess this will not do it, because keras is already imported. 
does **del np, del keras**, inside the function works? 
does **reload (np), reload (keras)** works ? 

**What is the correct way of doing this?** 

Thanks in advance.

**Comment by farmeh** [comment]
I tested. Above code works perfectly for me. No need to do anything else :)
